app/containers/CreateEventFeedback/template.js

```diff
@@ -0,0 +1,196 @@
+/* eslint quotes: 0 max-len: 0*/
+// @flow
+
+export type Question = {|
+  title: string,
+  labels?: string[],
+  type?: 'response' | '5scale' | 'interestScale' | 'posNegScale' | null,
+
+  // responses if type is 'response', otherwise datasets
+  responses?: Array<string>,
+  datasets?: Array<{
+    data: Array<any>,
+    backgroundColor: Array<string>,
+  }>,
+|}
+
+const oneToFive = ['1', '2', '3', '4', '5']
+
+const oneToFiveScale = {
+  labels: oneToFive,
+  type: '5scale',
+}
+
+const interestScale = {
+  labels: oneToFive,
+  type: 'interestScale',
+}
+
+const posNegScale = {
+  labels: oneToFive,
+  type: 'posNegScale',
+}
+
+const backgroundColor = ['#a9e0f2', '#1d6598', '#0f344e']
+
+const template: Question[] = [
+  {
+    title: 'How did the event impact your opinion about the company?',
+    labels: ['Positive impact', 'No change', 'Negative impact'],
+  },
+  {
+    title: 'How interested in working at this company are you now?',
+    ...interestScale,
+  },
+  {
+    title:
+      "How interested in writing your master's thesis at this company are you now?",
+    ...interestScale,
+  },
+  {
+    title: "Are you looking for a company to do your master's thesis at?",
+    labels: ['Yes, I am', 'No, I am not'],
+  },
+  {
+    title:
+      "Please motivate why you are or aren't interested in working at this company.",
+    type: 'response',
+  },
+  {
+    title: 'Do you feel qualified to work at this company?',
+    labels: ['Yes, I do', "No, I don't"],
+  },
+  {
+    title: 'How did you experience the atmosphere at the event?',
+    ...posNegScale,
+  },
+  {
+    title: 'Did you enjoy the activities at the event?',
+    ...oneToFiveScale,
+  },
+  {
+    title: 'Did you like the food at the event?',
+    ...oneToFiveScale,
+  },
+  {
+    title: 'What did you enjoy the most about the event and the company?',
+    type: 'response',
+  },
+  {
+    title: 'What could have been improved?',
+    type: 'response',
+  },
+]
+
+function scaleToYesNo(scaleData: Array<any> = []) {
+  let yes = 0
+  let no = 0
+  scaleData.forEach((amount, index) => {
+    if (index + 1 >= 3) {
+      yes += Number.parseInt(amount, 10)
```

Comment:
Don't think we need the `Number` prefix?

Reply:
Ah you're right, the ESLint config in some previous project I've worked on needed it, but ours doesn't. Will clean up.

Reply:
You can add that to the lint if you think it's a good change.